<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageRoiDialog</class>
 <widget class="QDialog" name="ImageRoiDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Regions of Interest (ROI)</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="vertical_layout" stretch="0,4,0,0,0,0,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pick one or two reference images (usually the first and last of an image series)</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontal_layout">
       <item>
        <widget class="ClickableLabel" name="reference_image1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>200</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>&lt;br&gt;&lt;br&gt;Click the Image button to pick</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ClickableLabel" name="reference_image2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>200</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>&lt;br&gt;&lt;br&gt;Click the Image button to pick</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Select Regions of Interest (ROI) drawing mode and related settings</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,2">
       <item>
        <layout class="QGridLayout" name="grid_layout" columnstretch="0,1,0,1,0,0,0,0,0" columnminimumwidth="1,1,0,0,1,0,0,0,0">
         <item row="1" column="3">
          <widget class="QSpinBox" name="rows_spinbox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="rows_label">
           <property name="text">
            <string>Rows</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="roi_placement_mode">
           <item>
            <property name="text">
             <string>Matrix</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Individual</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="roi_detection_mode">
           <item>
            <property name="text">
             <string>Partial</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cut to</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Largest</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QSpinBox" name="width_spinbox">
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="width_label">
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <spacer name="horizontal_spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="5">
          <widget class="QLabel" name="height_label">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QSpinBox" name="height_spinbox">
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="columns_label">
           <property name="text">
            <string>Columns</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="roi_shape">
           <item>
            <property name="text">
             <string>Circle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rectangle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ellipse</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Polygon</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="8">
          <widget class="QSpinBox" name="radius_spinbox">
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QLabel" name="radius_label">
           <property name="text">
            <string>Radius</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="columns_spinbox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Detection mode</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Placement mode</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Shape</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </item>
     <item>
      <widget class="QLabel" name="info_label">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="vertical_spacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="clear_button">
         <property name="text">
          <string>Clear ROIs</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_button">
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="done_button">
         <property name="text">
          <string>Done</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>ClickableLabel</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>done_button</sender>
   <signal>clicked()</signal>
   <receiver>ImageRoiDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>937</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
